lib/libze/libze.c

```diff
@@ -2057,6 +2105,15 @@ libze_destroy_cb(zfs_handle_t *zh, void *data) {
                                    ds);
         }
     }
+    cb.cb_target = zh;
+    cb.cbd = cbd;
+    cb.cb_nvl = fnvlist_alloc();
```

Comment:
don't shy away from whitespace in C. I'd put some empty lines to make code more readable 😍